client/src/Pages/RegisterPage/register.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import './register.css';
+import facebookIcon from './assets/facebook-icon.png';
+import googleIcon from './assets/google-icon.png';
+import Footer from '../../Components/Footer/footer';
+import NavBar from '../../Components/NavBar/navbar';
+import auctionAppLogo from '../../Components/NavBar/auction-app-logo.png';
+import { Link } from "react-router-dom";
+
+function Register() {
+  return (
+    <div className="login-container">
+      <NavBar />
+      <div className="container"><img className= "logo" src={auctionAppLogo} alt='Auction App Logo'></img></div>
+      <div className="login-content">
+        <div className="frame-180">
+          <div className="frame-178">
+            <h5 className="loginH1">REGISTER</h5>
+          </div>
+          <div className="frame-179">
+          <div className="frame-270">
+              <label htmlFor="email">First Name</label>
+              <input type="text" id="email" placeholder="John" />
+            </div>
+            <div className="frame-270">
+              <label htmlFor="email">Last Name </label>
+              <input type="text" id="email" placeholder="Doe" />
+            </div>
+            <div className="frame-270">
+              <label htmlFor="email">Enter Email</label>
+              <input type="email" id="email" placeholder="user@domain.com" />
+            </div>
+            <div className="frame-269">
+              <label htmlFor="password">Enter Password</label>
+              <input type="password" id="password" placeholder="*******" />
+            </div>
+            <div className="frame-271">
+              <Link to='/index' className="cta">Register</Link>
+            </div>
+          </div>
+          <div className="social-logins">
+            <button className="login-facebook">
+              <img src={facebookIcon} alt="Facebook Icon" />Signup with Facebook
+            </button>
+            <button className="login-gmail">
+              <img src={googleIcon} alt="Google Icon" />Signup with Gmail
```

Comment:
When sending the props could we try using this type of formatting 
```suggestion
              <img src={ googleIcon } alt="Google Icon" />Signup with Gmail
```